Pod/Classes/VOKCoreDataManager.m

```diff
@@ -524,6 +583,36 @@ - (NSManagedObjectContext *)temporaryContext
     return [self tempManagedObjectContext];
 }
 
+- (NSManagedObjectContext *)temporaryInMemoryContext
+{
+    //Spin up a new context
+    NSManagedObjectContext *context;
+    context = [[NSManagedObjectContext alloc] initWithConcurrencyType:NSPrivateQueueConcurrencyType];
+    
+    //Use the same MOM as the main context.
+    NSManagedObjectContext *mainContext = [[VOKCoreDataManager sharedInstance] managedObjectContext];
```

Comment:
I think this will barf if called off the main queue if the main context doesn't already exist since `initManagedObjectContext` 
Couldn't you get self.managedObjectModel directly in the line below?